bitnami/postgresql-ha/values.yaml

```diff
@@ -1141,7 +1141,7 @@ metrics:
   ##           usage: "GAUGE"
   ##           description: "Size of the database in bytes"
   ##
-  customMetrics: ""
+  customMetrics: {}
```

Comment:
Please, could you update also the README file? Just to have updated the values.yaml and also the README file.